packages/react/src/patterns/sub-patterns/Layout/Layout.js

```diff
@@ -22,29 +22,34 @@ const { prefix } = settings;
  */
 const _types = {
   '1-3': [`${prefix}--col-lg-4`, `${prefix}--col-lg-12`], // 1/4 - 3/4
+  '2-1': [`${prefix}--col-lg-8`, `${prefix}--col-lg-4`], // 2/3 - 1/3
 };
 
 /**
  * Returns the child classes with the proper column class names
  *
  * @param {string} type layout type
  * @param {number} stickyOffset Sticky offset amount (in pixels)
+ * @param {bool} border toggles border-bottom to the components
  * @param {object} children child nodes
  * @returns {*} modified child content
  * @private
  */
-function _updateChild(type, stickyOffset, children) {
+function _updateChild(type, stickyOffset, border, children) {
   const final = [];
 
-  children.map((child, i) => {
+  const componentArr = type === '2-1' ? [children[1], children[0]] : children;
```

Comment:
Can you help me understand what's happening here?

Reply:
The `Layout` uses the fist component as the size navigation, as the new proportion uses it on the right size, this ensures that the components are ordered properly.

Reply:
I think `Layout` doesn't really care what column is what, it's pretty generic. We can probably keep the order in which the columns come in, and Drupal team will need to do something like this:

```
<Layout>
  <div><ContentBlockSegment /></div>
  <div><LinkList /></div>
</Layout>
```